src/Header/AcceptHeaderItem.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Pavel Z
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Http\Header;
+
+/**
+ * Represents "Accept" header item.
+ */
+class AcceptHeaderItem
+{
+    /** @var string */
+    private $mime = null;
+
+    /** @var string */
+    private $type = null;
+
+    /** @var string */
+    private $subtype = null;
+
+    /** @var float */
+    private $quality = 1.0;
+
+    /** @var array */
+    private $params = [];
+
+    /**
+     * Parse accept header string.
+     *
+     * @param string $string
+     * @return static
+     */
+    public static function fromString(string $string): self
+    {
+        $elements = explode(';', $string);
+
+        $item = new static();
+        $item->setMime(trim(array_shift($elements)));
+
+        foreach ($elements as $element) {
+            [$name, $value] = explode('=', trim($element));
```

Comment:
if a chunk doesn't have `=` we will get `Undefined offset: 1` error. I propose to add a check for this case, at least we can throw a more informative exception with the message, that the header line is malformed